addons/fastroping/XEH_postInit.sqf

```diff
@@ -7,3 +7,12 @@
 [QGVAR(startFastRope), {
     [FUNC(fastRopeServerPFH), 0, _this] call CBA_fnc_addPerFrameHandler;
 }] call CBA_fnc_addEventHandler;
+
+["ACE3 Vehicles", QGVAR(cutRopes), localize LSTRING(Interaction_cutRopes), {
+    if ([vehicle ACE_player] call FUNC(canCutRopes)) then {
+        [vehicle ACE_player] call FUNC(cutRopes);
+        true
+    } else {
+        false
```

Comment:
Ugh, I was thinking of something and don't know what, doing too many things at once. :anguished: